infra/deploy-aio.sh

```diff
@@ -1,45 +1,56 @@
 #! /bin/bash
 
 export CLUSTER_NAME=$1
-export RESOURCE_GROUP=$2
+export STORAGE_ACCOUNT_NAME=$2
+export SCHEMA_REGISTRY_NAME=$3
+export RESOURCE_GROUP=$4
 export SUBSCRIPTION_ID=$(az account show --query id -o tsv)
-export LOCATION=$3
+export LOCATION=$5
 
 # register providers
 az provider register -n "Microsoft.ExtendedLocation"
 az provider register -n "Microsoft.Kubernetes"
 az provider register -n "Microsoft.KubernetesConfiguration"
-az provider register -n "Microsoft.IoTOperationsOrchestrator"
 az provider register -n "Microsoft.IoTOperations"
 az provider register -n "Microsoft.DeviceRegistry"
 
 # install CLI extensions
 echo "Installing CLI extensions..."
-az extension add --name connectedk8s
-az extension add --name azure-iot-ops
+curl -L -o connectedk8s-1.10.0-py2.py3-none-any.whl https://github.com/AzureArcForKubernetes/azure-cli-extensions/raw/refs/heads/connectedk8s/public/cli-extensions/connectedk8s-1.10.0-py2.py3-none-any.whl   
+az extension add --upgrade --source connectedk8s-1.10.0-py2.py3-none-any.whl --yes
+az extension add --upgrade --name azure-iot-ops --yes
 
 # create resource group
-if [ ! $(az group exists -n $RESOURCE_GROUP) ]; then
+if [ !$(az group exists -n $RESOURCE_GROUP) ]; then
     echo "Creating RG $RESOURCE_GROUP..."
     az group create --location $LOCATION --resource-group $RESOURCE_GROUP --subscription $SUBSCRIPTION_ID
 fi
 
 # connect arc cluster
 echo "Connecting cluster $CLUSTER_NAME..."
-az connectedk8s connect -n $CLUSTER_NAME -l $LOCATION -g $RESOURCE_GROUP --subscription $SUBSCRIPTION_ID
+az connectedk8s connect -n $CLUSTER_NAME -l $LOCATION -g $RESOURCE_GROUP --subscription $SUBSCRIPTION_ID --enable-oidc-issuer --enable-workload-identity
```

Comment:
`--enable-oidc-issuer --enable-workload-identity`
These settings are for usage of Secret Sync Controller and workload identity together with Key Vault but it's not implemented in this sample. Maybe remove or add a note why we are enabling this just for the cluster to be ready for further configuration.

Reply:
of course...that might confuse the user of this repo